src/scripts/script.js

```diff
@@ -38,179 +42,200 @@ const getCurrentAdditionalTemplate = () => {
+    ? getElementById("completedTasks")
+    : getElementById("currentTasks");
+
+  currentTodos.map(({ id, title, text, radio: priority }) => {
+    const todoLINode = document.createElement("li");
+    todoLINode.setAttribute(
+      "class",
+      `list-group-item d-flex w-100 mb-2 ${
+        !completedStatus ? "currentTodo" : ""
+      }`
+    );
+    todoLINode.setAttribute(
+      "style",
+      `background-color:${
+        !completedStatus
+          ? setTaskBackgroundColor(priority)
+          : setTaskBackgroundColor()
+      }`
+    );
+    todoLINode.setAttribute("id", `${id} `);
+    currentNode.append(todoLINode);
+    todoLINode.innerHTML = getTasksLITemplate(title, text, completedStatus);
+  });
+  if (!completedStatus) {
+    const map_removeButton = document.querySelectorAll(".btn-danger");
+    setClicksOnNode(map_removeButton, removeToDoItem);
+
+    const map_editButton = document.querySelectorAll(".btn-info");
+    setClicksOnNode(map_editButton, editTodoTask);
+
+    const map_completeButton = document.querySelectorAll(".btn-success");
+    setClicksOnNode(map_completeButton, completeTodoTask);
+  }
+  setCountTodos();
+};
 
 const addTask = () => {
-    const btn_saveTask = getElementById("form-group-save-task");
-    $('#exampleModal').modal("show");
-    btn_saveTask.addEventListener("click", () => {
-        const id_dateValue = +new Date;
-        addTodoToLocalStorage(id_dateValue, id_dateValue, ...getModalValues(), false);
+  const btn_saveTask = getElementById("form-group-save-task");
+  $("#exampleModal").modal("show");
+  btn_saveTask.addEventListener("click", () => {
+    const id_dateValue = +new Date();
+    addTodoToLocalStorage(
+      id_dateValue,
+      id_dateValue,
+      ...getModalValues(),
+      false
+    );
+  });
+};
+function toggleStyle() {
+  const fontElement = document.querySelectorAll("h3");
+  if (this.checked) {
+    document.getElementsByTagName("body")[0].style.backgroundColor = "#000";
+    fontElement.forEach((felement) => {
+      felement.style.color = "#fff";
+    });
+  } else {
+    document.getElementsByTagName("body")[0].style.backgroundColor = "#fff";
+    fontElement.forEach((felement) => {
+      felement.style.color = "#000";
     });
+  }
 }
-
-
-
-
 const sortTodoList = (compare) => {
-    const currentContainer = getElementById("currentTasks");
-    const compareCore = (compare == "sort-from-new") ? true : false;
-    [...currentContainer.children]
-        .sort((current, next) => {
-            if (current.id > next.id) {
-                return compareCore ? 1 : -1;
-            }
-            if (current.id < next.id) {
-                return !compareCore ? 1 : -1;
-            }
-            if (current.id == next.id) {
-                return 0;
-            }
-        }).forEach(node => currentContainer.appendChild(node));
+  const currentContainer = getElementById("currentTasks");
+  const compareCore = compare === "sort-from-new" ? true : false;
```

Comment:
it's the same `const compareCore = compare === "sort-from-new"`

Reply:
fixed